### PR TITLE
[IMP] web: start loading data before user click menu

### PR DESCRIPTION
**Pull request by yelizariev**
dirty implementation of the idea to record a preview

### CODE REVIEW

**Comment by yelizariev** [comment]
WIP

File: addons/web/static/src/core/network/rpc_service.js [hunk 1]
```diff
@@ -95,11 +95,35 @@ function jsonrpc(env, rpcId, url, params, settings = {}) {
...
+        hash |= 0; // Convert to 32bit integer
+    }
+    return hash;
+};
 export const rpcService = {
     async: true,
     start(env) {
         let rpcId = 0;
         return function rpc(route, params = {}, settings) {
+            if (route == "/web/dataset/search_read" && window.localStorage['fast-menu-opening-proof-of-concept']) {
```
Comment (ged-odoo) [inline addons/web/static/src/core/network/rpc_service.js:113]: are you sure about this? it looks very wrong to put records data in a cache. also, this PR looks non trivial, don't hesitate to come in rd-framework-js to explain what you are trying to do, so we can help you

Comment (yelizariev) [reply addons/web/static/src/core/network/rpc_service.js:113]: This is supposed to be short term cache (few seconds)

**Comment by yelizariev** [comment]
https://www.odoo.com/web#id=2612908&action=333&active_id=133&model=project.task&view_type=form&cids=1&menu_id=4720

**Comment by yelizariev** [comment]
cancel for now because of complexity it introduces

**Pull request closed without merging**